frontend/src/components/ConfigForm.vue

```diff
@@ -6,35 +6,71 @@
     </v-stepper-step>
 
     <v-stepper-content step="1">
-      <v-text-field
-        label="Application Acronym"
-        required
-        :value="userAppCfg.applicationAcronym"
-        v-on:keyup.stop="updateAppCfgField('applicationAcronym', $event.target.value)"
-      ></v-text-field>
-      <v-text-field
-        label="Application Name"
-        required
-        :value="userAppCfg.applicationName"
-        v-on:keyup.stop="updateAppCfgField('applicationName', $event.target.value)"
-      ></v-text-field>
-      <v-text-field
-        label="Application Description"
-        required
-        :value="userAppCfg.applicationDescription"
-        v-on:keyup.stop="updateAppCfgField('applicationDescription', $event.target.value)"
-      ></v-text-field>
-      <v-select
-        :items="commonServices"
-        label="Common Service Required"
-        multiple
-        chips
-        deletable-chips
-        :value="userAppCfg.commonServices"
-        v-on:change="updateAppCfgField('commonServices', $event)"
-      ></v-select>
+      <v-form v-model="step1Valid">
+        <v-layout row wrap>
+          <v-flex xs12 md7>
+            <v-text-field
+              label="Application Acronym"
+              required
+              :value="userAppCfg.applicationAcronym"
+              v-on:keyup.stop="updateAppCfgField('applicationAcronym', $event.target.value)"
+              :counter="30"
```

Comment:
Wondering if we should be making the counter a variable instead of a magic number across the codebase